client/layout/html-is-iframe-classname.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import { useEffect, memo } from 'react';
+import { useSelector } from 'react-redux';
+import debugFactory from 'debug';
+
+/**
+ * Internal dependencies
+ */
+import isIframeForHtmlElement from 'state/selectors/is-iframe-for-html-element';
+
+const debug = debugFactory( 'calypso:layout:html-is-iframe-classname' );
+
+const HtmlIsIframeClassname = memo( () => {
```

Comment:
Instead of `memo`, I'd add a dependency parameter to `useEffect`:
```js
const HtmlIsIframeClassname = () => {
  const isIframe = useSelector( isIframeForHtmlElement );
  useEffect( () => {
    // add and remove classes
  }, [ isIframe ] );

  return null;
}
```
That's a more common way to run an effect only when something (i.e., `isIframe` value) changes and do nothing otherwise. `memo` should also work 100%, but it's not exactly the ideal intended usage of it.